src/reducers/index.js

```diff
@@ -37,6 +49,14 @@ const cardReducer = (state = initialState, action) => {
       return Object.assign({}, state, action.payload);
     case LOAD_USERS:
       return Object.assign({}, state, { users: action.payload.users });
+    case LOAD_LOGIN:
+      return Object.assign({}, state, { loggedIn: localStorage.getItem('logged_in'), userId: localStorage.getItem('userId') });
```

Comment:
It's best to retrieve the localStorage values at the top of the file and outside the reducer itself. Then you can make it part of the initial state.